GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -225,15 +225,29 @@ protected override void Dispose(bool disposing)
         {
             if (disposing)
             {
+                _superprojectCurrentCheckout = null;
+                _selectedObjectIds = null;
+                _baseCommitToCompare = null;
                 _revisionSubscription?.Dispose();
+                _revisionSubscription = null;
                 _revisionReader?.Dispose();
+                _revisionReader = null;
```

Comment:
I strongly prefer not setting these fields to `null` unless absolutely required. It makes reasoning about the state of fields more confusing, and does not play well with Nullable Reference Types.

Reply:
I've seen other `Dispose` and `TearDown` setting fields to `null`. That's why I tried the same although the GC should do the job.
I must confess I am fond of C++ smart pointers and feel very uncofortable with the drawbacks of GC, especially regarding fields and regarding exceptions in `Dispose`. But I must take .Net as it is.

Reply:
Removed from modified `Dispose` and from all `[OneTime]TearDown`.